Makefile

```diff
@@ -8,6 +8,8 @@ IMAGE_BUILD_CMD ?= docker build
 IMAGE_BUILD_EXTRA_OPTS ?=
 IMAGE_PUSH_CMD ?= docker push
 CONTAINER_RUN_CMD ?= docker run
+BASE_IMG_FULL ?= debian:buster-slim
+BASE_IMG_MINIMAL ?= gcr.io/distroless/base
```

Comment:
nit: I think `BASE_IMAGE_FULL` and `BASE_IMAGE_MINIMAL` (without the abbreviation) would be more in line with the other variables